tech/languages/ruby/ruby-installation.md

```diff
@@ -61,48 +49,46 @@ RUBYPICK=_jruby_
 
 The first step is to install dependencies for Ruby.
 
-```
-sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
+```console
+$ sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
 ```
 
-Install rbenv
+Then we are going to clone rbenv and the ruby-build rbenv plugin into the home directory.
+To make rbenv command available we append your shell's rc file with initialization.
 
+```console
+$ git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+$ git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+$ echo 'eval "$(~/.rbenv/bin/rbenv init -)"' >> ~/.bashrc
 ```
-cd
-git clone https://github.com/rbenv/rbenv.git ~/.rbenv
-echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
-echo 'eval "$(rbenv init -)"' >> ~/.bashrc
-source ~/.bashrc
-exec $SHELL
-
-git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
-echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
-source ~/.bashrc
-exec $SHELL
-```
-
-Install Ruby
 
+Next step we have to refresh the shell to make the binaries available. This can be achieved
+by either the following command or opening a new terminal:
+```console
+$ exec $SHELL
```

Comment:
I do not think this will work without the sourcing of bashrc, but feel free to try.

Reply:
It seems to just work. However, thanks to retrying this I found 2 things: Perl is required and there is rbenv RPM package in Fedora.

Reply:
It works.